internal/pkg/convert/graph.go

```diff
@@ -124,7 +124,6 @@ func (graph *GraphLLB) buildLocalContext() {
 		llb.ExcludePatterns(
 			[]string{
 				"**/.*",
-				"**/" + constants.PkgYaml,
```

Comment:
do we need this, or should we inject the package name as the environment variable to the build?

Reply:
hmm that works too, let me see if it's easier

Reply:
fixed